custom_components/luxtronik/const.py

```diff
@@ -394,6 +394,7 @@ class LuxParameter(StrEnum):
     # ? P1138_SWIMMING_POOL_ENERGY_INPUT: Final = "parameters.Unknown_Parameter_1138" -->
     P1139_COOLING_ENERGY_INPUT: Final = "parameters.Unknown_Parameter_1139"
     P1140_SECOND_HEAT_GENERATOR_AMOUNT_COUNTER: Final = "parameters.Unknown_Parameter_1140"
+    P1148_HEATING_TARGET_TEMP_RBE: Final = "parameters.Unknown_Parameter_1148"
```

Comment:
RBE ist german for "Raum Bedien Einheit". Can we call it "ROOM_THERMOSTAT"?
--> P1148_HEATING_TARGET_TEMP_ROOM_THERMOSTAT